File: easy/fizzbuzz.py
import sys


if __name__ == '__main__':
    f = open(sys.argv[1])
    for l in f:
        a, b, n = list(int(x) for x in l.split(" "))
        out = []
        for x in range(1, n + 1):
            out.append('FB' if x % a is 0 and x % b is 0
                       else 'F' if x % a is 0
                       else 'B' if x % b is 0
                       else str(x))
        print ' '.join(out)

File: easy/fizzbuzz.rb
File.readlines(ARGV[0]).each do |l|
    l.strip!
    a, b, n = l.split
    out = []
    (1..n.to_i).each do |x|
        if x % a.to_i == 0 and x % b.to_i == 0
            out.push('FB')
        elsif x % a.to_i == 0
            out.push('F')
        elsif x % b.to_i == 0
            out.push('B')
        else
            out.push(x.to_s)
        end
    end
    puts out.join(' ')
end
